megamek/src/megamek/common/FixedWingSupport.java

```diff
@@ -294,6 +299,202 @@ public void addBattleForceSpecialAbilities(Map<BattleForceSPA,Integer> specialAb
         }
     }
 
+    @Override
```

Comment:
This was copied from Tank and edited to remove anything irrelevant to fixed wing, airship, or satellite support vees.